crates/wasi/src/bindings.rs

```diff
@@ -2,26 +2,27 @@
 //
 // Note that this is only done for interfaces which can block, or those which
 // have some functions in `only_imports` below for being async.
-pub mod sync_io {
-    pub(crate) mod _internal {
+pub mod sync {
```

Comment:
I'm unsure about `sync` vs. `async_io`. It seems like the previous `sync_io` contrasts more clearly with `async_io`

Reply:
Ah the `async_io` name doesn't show up at all, that's just to contain everything and then reexport only the bits we need. The only modules here are `bindings` and `bindings::sync`, but these bindings are relatively rare to access since they're the raw traits/types that most embedders won't be working with.